serve.js

```diff
@@ -57,9 +51,9 @@ function runTasks(cb) {
     //then set the interval
     if (!intervalSet) {
       intervalSet = true;
-      var interval = 6 * (5 * 1e4);
+      var interval = 6 * (10 * 1e4);
```

Comment:
I had it set lower for testing purposes - 10 minutes is more than ample, api-sync updates jsdelivr in real time based on a webhook (a feature we should add to the api) but all other cdn's update on the half hour.